### DISCUSSION TITLE
If Multiline Replies is disable, line break should be used as Stop sequence trigger word like "Your name"

### DISCUSSION THREAD

**Discussion by Myrminki** [LostRuins/koboldcpp#83] [Ideas]
and being able to add additional trigger words manually in case the bot bypasses the stop sequence by changing the username in the self replies, eg 'user:' to 'user :' as gpt4all usually does, well in my case it does that

**Comment by LostRuins** [comment]
Customizable stop sequences from the UI is eventually planned, but for now you can set the "Chat Name" inside the Lite UI and the AI will use that as the stop sequence.

**Reply by Myrminki** [reply]
Thank you very much for the feature, it is a shame only 1 more trigger can be added, but in the meantime I can use "'\n" and it works great. I hope that later "," can be used to define multiple triggers

**Reply by LostRuins** [reply]
Note that it will add on to the existing stop sequences already in chat (e.g. You:), which will still be active.

**Reply by Myrminki** [reply]
Actually "\n" does not even let "you:" trigger do anything because the bots usually use "\n" when their are about to reply to themself this trigger alone just fixes almost all the problems i had when chatting and that's why i asked it to be a default trigger when multi line replies are disabled  

The multi-trigger will be useful to prevent the bot from going into a brainstorm when multi line responses are enabled

**Reply by Myrminki** [reply]
or when multi line reply is disable replace "you:" with "\n"

**Reply by Myrminki** [reply]
"stop_sequence": ["Myr:", "\n"]}

Processing Prompt (32 / 32 tokens)
Generating (70 / 80 tokens)
(Stop sequence triggered: <
>)

**Reply by Myrminki** [reply]
I take back what I said, there are cases where the bot will ignore the "\n" and trigger with the name, so it's fine as it is now for chat mode.
